### PR TITLE
Remove iOS code

### PR DESCRIPTION
**Pull request by atavism**

### CODE REVIEW

**Comment by jigar-f** [comment]
@atavism even after merging this, old ios client will still work correct?

**Comment by atavism** [comment]
> @atavism even after merging this, old ios client will still work correct?

No.. do we need to support it? If we need to add back the ios code to flashlight for any reason, we can always reference this PR

**Comment by jigar-f** [comment]
Even if we upload this build to production we still meed support for old clients, till most of user update app to latest

**Comment by atavism** [comment]
> Even if we upload this build to production we still meed support for old clients, till most of user update app to latest

@jigar-f Oh, I see what you mean. Old versions of the app won't stop working (those builds will still be using previous versions of flashlight that included the iOS client code). So we should be fine merging this

**Comment by jigar-f** [comment]
@atavism Thanks for letting me know, I will merge this shortly.

**Comment by jigar-f** [comment]
@atavism Here are the changes-: https://github.com/getlantern/flashlight/pull/1362/commits/8bd9703242d23e89d1571503f42d4f81683cccb7 If that works for you.

File: bandit/bandit.go [hunk 1]
```diff
@@ -34,6 +30,9 @@ type BanditDialer struct {
 	dialers      []Dialer
 	bandit       *bandit.EpsilonGreedy
 	statsTracker stats.Tracker
+	// lastDialCountryCode stores the last dialed country code.
+	// This is used to avoid unnecessary calls to the UI thread
+	lastDialCountryCode atomic.Value
```
Comment (jigar-f) [inline bandit/bandit.go]: Could you take a look at it here, Does this break any other code?
cc @atavism @reflog

File: bandit/bandit.go [hunk 2]
```diff
@@ -160,6 +163,13 @@ func differentArm(existingArm, numDialers int) int {
 
 func (o *BanditDialer) onSuccess(dialer Dialer) {
 	countryCode, country, city := dialer.Location()
+	lastCountryCode, _ := o.lastDialCountryCode.Load().(string)
+	if lastCountryCode != "" && lastCountryCode == countryCode {
+		log.Debugf("Same country code as last returning")
```
Comment (reflog) [inline bandit/bandit.go]: let's remove this log. too noisy

**Review by reflog — APPROVED** [review]
👍🏻  (except for the log comment)

File: bandit/bandit.go (hunk 2 above)
Comment (jigar-f) [reply bandit/bandit.go]: Thanks, Done.

**Pull request merged**